src/drivers/ins/microstrain/module.yaml

```diff
@@ -0,0 +1,257 @@
+module_name: MICROSTRAIN
+parameters:
+    - group: Sensors
+      definitions:
+
+        MS_MODE:
+            description:
+                short: Toggles using the device as the primary EKF
+                long: |
+                    Setting to 1 will publish data from the device to the vehicle topics (global_position, attitude, local_position, odometry), estimator_status and sensor_selection
+                    Setting to 0 will publish data from the device to the external_ins topics (global position, attitude, local position)
+                    Restart Required
+            type: int32
+            default: 1
+
+        MS_IMU_RATE_HZ:
+            description:
+                short: IMU Data Rate
+                long: |
+                    IMU (Accelerometer and Gyroscope) data rate
+                    The INS driver will be scheduled at a rate 2*MS_IMU_RATE_HZ
+                    Max Limit: 1000
+                    0 - Disable IMU datastream
+                    The max limit should be divisible by the rate
+                    Restart required
+            type: int32
+            default: 500
+
+        MS_MAG_RATE_HZ:
+            description:
+                short: Magnetometer Data Rate
+                long: |
+                    Magnetometer data rate
+                    Max Limit: 1000
+                    0 - Disable magnetometer datastream
+                    The max limit should be divisible by the rate
+                    Restart required
+            type: int32
+            default: 50
+
+        MS_BARO_RATE_HZ:
+            description:
+                short: Barometer data rate
+                long: |
+                    Barometer data rate
+                    Max Limit: 1000
+                    0 - Disable barometer datastream
+                    The max limit should be divisible by the rate
+                    Restart required
+            type: int32
+            default: 50
+
+        MS_FILT_RATE_HZ:
+            description:
+                short: EKF data Rate
+                long: |
+                    EKF data rate
+                    Max Limit: 1000
+                    The max limit should be divisible by the rate
+                    0 - Disable EKF datastream
+                    Restart required
+            type: int32
+            default: 250
+
+        MS_GNSS_RATE_HZ:
+            description:
+                short: GNSS data Rate
+                long: |
+                    GNSS receiver 1 and 2 data rate
```

Comment:
Might be worth noting that some devices support different rates here. I think this is the only one with a problem, but GQ7 only supports a max data rate of 2 hertz